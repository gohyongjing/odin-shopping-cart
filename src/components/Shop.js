import React, {useState} from 'react';
import ShopHeader from './ShopHeader';

const Shop = () => {


    const [cart, setCart] = useState({});

    return (
      <div className='shop'>
        <ShopHeader cart={cart} />
      </div>
    );
  };
  
  export default Shop;