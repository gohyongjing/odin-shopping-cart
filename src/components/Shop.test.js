import React from 'react';
import { render } from "@testing-library/react";
import Shop from './Shop';

describe("Shop component", () => {
    it("renders", () => {
        const { asFragment } = render(<Shop />);
        expect(asFragment()).toMatchSnapshot();
    });

    it
});

