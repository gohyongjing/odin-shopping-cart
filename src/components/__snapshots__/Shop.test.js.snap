// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shop component renders 1`] = `
<DocumentFragment>
  <div
    class="shop"
  >
    <div
      class="shop-header"
    >
      <span>
        Cart items: 0
      </span>
      <button>
        Check out and pay
      </button>
    </div>
    <div
      class="shop-item"
    >
      <h2>
        Elixir Collector
      </h2>
      <img
        alt="Elixir Collector"
        src="ElixirCollectorCard.webp"
      />
      <div>
        Price: $60
      </div>
      <input
        type="number"
        value="0"
      />
    </div>
    <div
      class="shop-item"
    >
      <h2>
        Level 10 Wall
      </h2>
      <img
        alt="Level 10 Wall"
        src="Wall10.webp"
      />
      <div>
        Price: $20
      </div>
      <input
        type="number"
        value="0"
      />
    </div>
    <div
      class="shop-item"
    >
      <h2>
        Phantom
      </h2>
      <img
        alt="Phantom"
        src="Phantom.webp"
      />
      <div>
        Price: $29
      </div>
      <input
        type="number"
        value="0"
      />
    </div>
    <div
      class="shop-item"
    >
      <h2>
        Rabadon's Deathcap
      </h2>
      <img
        alt="Rabadon's Deathcap"
        src="Rabadons_Deathcap.webp"
      />
      <div>
        Price: $36
      </div>
      <input
        type="number"
        value="0"
      />
    </div>
  </div>
</DocumentFragment>
`;
